components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -47,6 +47,8 @@ class SystemEngineSession(private val defaultSettings: Settings? = null) : Engin
     @Volatile internal var trackingProtectionPolicy: TrackingProtectionPolicy? = null
     @Volatile internal var webFontsEnabled = true
     @Volatile internal var internalSettings: Settings? = null
+    internal var alertJSCount = 0
```

Comment:
Probably needs `@Volatile` for these attributes as well.

Reply:
I had it locally :P, didn't notice that I didn't pushed it